tests/software/multiqc/test.yml

```diff
@@ -0,0 +1,9 @@
+- name: Run multiqc test workflow
+  command: nextflow run ./tests/software/multiqc/ -profile docker -entry test_multiqc -c tests/config/nextflow.config
+  tags:
+    - multiqc
+  files:
+    - path: output/test_multiqc/test_fastqc.html
+      md5sum: 7027e72c3e55292c1567f12e02565e3b
+    - path: output/test_multiqc/test_fastqc.html
```

Comment:
@Emiller88 ,  could you please help me out a bit?

I'd like to understand how to obtain these output via `pytest`   and then update these here.

Reply:
I don't have a fancy way of doing it, I just put in a bad md5sum, and it spits out what it should have been, then I copy that md5 sum into where it should have been. I think I'll come up with a better workflow with nnn. 

Also, I think the htmls may need to just be "includes" or whatever they call it and pick some values.

Reply:
For the time being, I've removed the `md5sum`.